plasmapy/physics/transport/tests/test_collisions.py

```diff
@@ -183,13 +183,13 @@ def test_GMS1_negative(self):
         Murillo, and Schlanges PRE (2002). This checks for when
         a negative (invalid) Coulomb logarithm is returned.
         """
-        with pytest.raises(exceptions.PhysicsError, match="< 0"):
-            methodVal = Coulomb_logarithm(self.temperature2,
-                                          self.density2,
-                                          self.particles,
-                                          z_mean=np.nan * u.dimensionless_unscaled,
-                                          V=np.nan * u.m / u.s,
-                                          method="GMS-1")
+        with pytest.warns(exceptions.CouplingWarning, match="relies on weak coupling"):
```

Comment:
I have not used the `match` keyword in `pytest.warns`, but I should start doing so!